src/migration/v1_10_5.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from pymongo import UpdateOne
+
+from conf import DEFAULT_LOGGER
+from lib import MongoCustomClient
+from lib.util import query
+
+_LOGGER = logging.getLogger(DEFAULT_LOGGER)
+
+
+@query
+def monitoring_alert_number_remove_collection(mongo_client: MongoCustomClient):
+    mongo_client.drop_collection('MONITORING', 'alert_number')
+
+
+@query
+def monitoring_alert_refactor_alert_number_by_domain_id(mongo_client: MongoCustomClient):
+    pipelines = [
```

Comment:
For the logic of querying `domain_id`, `distinct` rather than `aggregate` is recommended.